docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,255 @@
+If the measured parameter drops below the low end or goes above the high end, a fault flag is raised, and can be viewed in the ``fault_now`` parameter. 
+The ``fault_now`` parameter is only set by hardware. All faults that have occurred since the module's start up can be viewed in the ``fault_ever`` parameter. 
+The value of ``fault_ever`` is generally set by the hardware and cleared by the user.
+
+The module's behavior when faults are detected is determined by the ``fault_latching`` parameter. If ``fault_latching`` is set to 1, then any high flags 
+in ``fault_ever`` place the module into safe mode. In safe mode, the module is set to coast, and it will not spin or apply any control voltage. The module remains in safe mode until the user clears ``fault_ever``. 
+When ``fault_latching`` is set to 0, the module remains in safe mode until all power safety parameters are back in their safe operating range, and ``fault_now`` returns to 0.
+
+Suppose the high limit of power safety's microcontroller temperature parameter is 115°C. If your module's microcontroller ever reaches 115°C, the module enters 
+safe mode, and will not spin until it exits safe mode. If ``fault_latching`` is set to 0, then once the module's microcontroller temperature drops below 115°C, 
+it will exit safe mode, and be able to spin once again. If ``fault_latching`` is 1, then your module will not exit safe mode until its microcontroller temperature 
+drops below 115°C and ``falt_ever`` is reset to 0.
+
+Internal Programmable Voltage Detector (PVD)
+------------------------------------------------
+Each module's microcontroller has an internal PVD that monitors the microcontroller's Vdd voltage level. Vdd is the voltage used to power the microcontroller, and if 
+it drops too low, your module's microcontroller cannot function properly. In the event that Vdd drops below the pre-programmed threshold value (~2.8V), the module is forced to stop spinning. 
+The module will only be allowed to resume normal operation when Vdd has risen back above the voltage threshold (~2.9V).
+
+.. _slew_rate:
+
+Slew Rate Limiter
+==================
+Your module's slew rate limit defines the maximum change in control voltage per second. The slew rate limiter can be used to limit how quickly your module attempts to 
+change speed, which can be useful to protect external electronics or improve integration with flight controllers in certain cases.
+
+There are two parameters in the :ref:`Brushless Drive client <brushless_drive>` that affect your module's slew rate limiting:
+
+* ``slew_volts_per_second`` defines the actual slew rate in volts per second applied to voltage commands. This defines the strength of the filter's impact.
+* ``slew_enable`` determines whether or not ``slew_volts_per_second`` is actually applied to voltage commands. If ``slew_enable`` is false, then there will not be slew limiting applied.
```

Comment:
I think these slew guys are also available through the Control Center, may be worth showing that they can be configured through there as well, just showing that it exists at least